transforms/universal/ededup/python/README.md

```diff
@@ -12,7 +15,11 @@ This is a python version of ededup
 The set of dictionary keys holding [EdedupPythonTransform](src/ededup_transform_python.py) 
 configuration for values are as follows:
 
-* _ededup_doc_column_ - specifies the name of the column containing a document
+* _doc_column_ - specifies the name of the column containing a document
```

Comment:
defaults should be specified as well.  perhaps follow the table idea from here https://github.com/IBM/data-prep-kit/tree/dev/transforms/language/pdf2parquet/python#parameters